rosetta-source/src/main/rosetta/observable-asset-calculatedrate-func.rosetta

```diff
@@ -32,7 +32,7 @@ func EvaluateCalculatedRate: <"Evaluate a calculated rate as described in the 20
         results FloatingRateSettingDetails (1..1) <"detailed results of the floating rate calculation.">
 
     // get the FRO
-    alias fro: floatingRateOption
+    alias fro: interestRateIndex
```

Comment:
Adjust the alias name and the comment referring to "FRO"

Reply:
updated to `alias rate: interestRateIndex` 
references to `fro` in the function updated to `rate`